langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaOpenAiChatModelIT.java

```diff
@@ -50,5 +56,123 @@ void should_generate_response() {
         assertThat(response.finishReason()).isEqualTo(STOP);
     }
 
+    @Test
+    void should_listen_request_and_response() {
```

Comment:
I am not sure these tests are required, it is already tested in `OpenAiChatModelIT`

Reply:
Maybe it's no need to test it.